drake/solvers/Constraint.h

```diff
@@ -116,12 +116,12 @@ class LinearConstraint : public Constraint {
   virtual void eval(const Eigen::Ref<const Eigen::VectorXd>& x,
                     Eigen::VectorXd& y) const override {
     y.resize(getNumConstraints());
-    y = getMatrix() * x;
+    y = getMatrix() * x * -1;
   }
   virtual void eval(const Eigen::Ref<const TaylorVecXd>& x,
                     TaylorVecXd& y) const override {
     y.resize(getNumConstraints());
-    y = getMatrix().cast<TaylorVarXd>() * x;
+    y = getMatrix().cast<TaylorVarXd>() * x * -1;
```

Comment:
Actually, this multiply was totally wrong.  I found the problem while implementing the lower bounds tests, and I've removed it.